### DISCUSSION TITLE
How to update serverpod installed from git

### DISCUSSION THREAD

**Discussion by y3l1n4ung** [serverpod/serverpod#1323] [Q&A] [answered]
I installed the ServerPod from Git. Is 'git pull origin main' sufficient for updating Serverpod?

**Comment by yahu1031** [comment]
Yes it does and better if you fork it and disable the upstream. So that you could stay safe from pushing any local changes.

**Reply by y3l1n4ung** [reply]
Let's say I initially installed Serverpod 0.9 from Git, and then I updated it to version 1.1.1 using git pull origin main. Is there a need to run `pub get` to update dependencies as well?

**Reply by yahu1031** [reply]
Yes, Getting pull will not update the cli directly. So, You need to `pub get` and create a cli again.

**Comment by y3l1n4ung** [comment, marked as answer]
I need to run `util/pub_get_all` from serverpod root dir.
